Providers/Modules/Plugins/Common/plugin/out_oms_blob.rb

```diff
@@ -71,6 +71,11 @@ def create_blob_put_request(uri, msg, request_id, file_path = nil)
         headers[OMS::CaseSensitiveString.new("x-ms-AzureResourceId")] = azure_resource_id
       end
       
+      azure_region = OMS::Configuration.azure_region if define?(OMS::Configuration.azure_region)
```

Comment:
do we need to update out_oms_api.rb as well?

Reply:
i do not see that file in DSC modules.

Reply:
oh, thats weird :)